Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM ubuntu:latest
-RUN apt-get update -y
+RUN apt-get update
+RUN apt-get upgrade -y
```

Comment:
does upgrade is needed, as it will increase size of image

Reply:
OKay removed it